src/components/RestaurantHeader/RestaurantHeader.tsx

```diff
@@ -3,6 +3,8 @@ import { ImageBackground, StyleSheet } from "react-native"
 import Typography from "../Typography"
 import Box from "../Box"
 
+const ASSETS_URL = process.env.PMO_ASSETS
```

Comment:
I wouldn't make these capital, as it still says something about the source. We don't care where the value comes from, only that we have them.